dpe/src/support.rs

```diff
@@ -161,6 +171,7 @@ pub mod test {
                 | (1 << 22)
                 | (1 << 21)
                 | (1 << 19)
+                | (1 << 18)
```

Comment:
nit: remove this flag given that `return-certificate` is implicit.

Reply:
oh jk. 18 is cdi export

Reply:
Closing this, as I think this is resolved but please re-open if I am mistaken.